### PR TITLE
TFKUBE-300: Release 2.0.0

### PR DESCRIPTION
**Pull request by yzha645**
## Pull request description

This PR releases new version 2.0.0. 

## Checklist
- [x] I have successful end to end tests run (with & without domain)
- [ ] I have added unit tests (if applicable)
- [ ] I have user documentation (if applicable)

### CODE REVIEW

**Comment by yzha645** [comment]
e2e test: 
https://github.com/atlassian-labs/data-center-terraform/runs/6462983600
https://github.com/atlassian-labs/data-center-terraform/runs/6462985753

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 2.0.0
+
+**Release date:** 2022-05-17
+
+* Added support for Jira Software, Jira Service Management, Confluence and Bitbucket
+* Added feature to pre-populate products with existing dataset
```
Comment (errcode1202) [inline CHANGELOG.md:8]: AWS EBS and RDS snapshots can now be used for restoring datasets
```suggestion
* AWS EBS and RDS snapshots can now be used for restoring datasets
```

**Review by errcode1202 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+* Changed shared home storage from EFS to NFS for all products
+* Made product version configurable
+* Added autoscaler to EKS cluster
+* Bumped up Helm chart version
```
Comment (jjeongatl) [inline CHANGELOG.md:12]:
```suggestion
* Upgraded Helm chart version to 1.3.0
```

**Review by jjeongatl — APPROVED** [review]

**Review by nanux — APPROVED** [review]
I think we can make it explicit that an upgrade from the pre 2.x version is not supported.

**Pull request merged**